app/models/subscriber.rb

```diff
@@ -0,0 +1,4 @@
+class Subscriber < ApplicationRecord
+    has_many :subscriptions
+     has_many :magazines, through: :subscriptions
```

Comment:
nit: try to keep indentation consistent!
```suggestion
    has_many :magazines, through: :subscriptions
```